internal/command/registry/command.go

```diff
@@ -0,0 +1,24 @@
+package registry
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/superfly/flyctl/internal/command"
+)
+
+func New() *cobra.Command {
+	const (
+		usage = "registry"
+		short = "Operate on registry images"
+		long  = "Scan registry images for an SBOM or vulnerabilities."
+	)
+	cmd := command.New(usage, short, long, nil)
```

Comment:
To make it hidden, if that's the route we want to go for letting people test it

```suggestion
	cmd := command.New(usage, short, long, nil)
	cmd.Hidden = true
```

Reply:
thank you. done.